tests/unit/strategies_test.py

```diff
@@ -0,0 +1,95 @@
+import operator
+
+import pytest
+
+from pydeepmerge import deep_merge
+from pydeepmerge.strategies import meta_operator_merge
+
+
+def return_right(left, right):
```

Comment:
nitpick: could be turned into a shared test fixture

Added #6 to address it later